src/net.cpp

```diff
@@ -2445,8 +2445,11 @@ CNode::~CNode()
 
 void CNode::AskFor(const CInv& inv)
 {
-    if (mapAskFor.size() > MAPASKFOR_MAX_SZ || setAskFor.size() > SETASKFOR_MAX_SZ)
+    if (mapAskFor.size() > MAPASKFOR_MAX_SZ || setAskFor.size() > SETASKFOR_MAX_SZ) {
+        LogPrintf("CNode::AskFor -- Buffers full: mapAskFor.size = %d, setAskFor.size = %d, MAPASKFOR_MAX_SZ = %d, SETASKFOR_MAX_SZ = %d\n",
```

Comment:
Not exactly sure what to call it, since I don't fully understand what setAskFor is even for.

Will add node id.